src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,220 @@
+The AWSCloudWatchLoggingPlugin log levels can be configured remotely.  When configured, the Amplify Logger will fetch the remote log levels, defined in a `remoteloggingconstraints.json` file and deployed to an S3 bucket, via an API Gateway endpoint that is integrated with a lambda.  The log levels defined in the remote logging constraints file will overwrite the local log levels and will persist across app sessions.
+
+You will need to provision and maintain your own backend resources.  The following provides a CDK construct example to provision the required AWS resources.
+
+<Callout warning>
+When you are provisioning your backend resources, you are responsible for following best security practices and securing your backend resources.
+</Callout>
+
+## CDK construct
+Below is an example CDK construct that provisions the Amazon CloudWatch, AWS API Gateway, AWS Lambda, and AWS S3 bucket.  The CDK construct also deploys the `remoteloggingconstraints.json` file to the AWS S3 bucket. 
+
+### Permissions
+The CDK construct sample created IAM permission policies and assigned it to the Amplify authenticated and unauthenticate role.  
+1. The API Gateway endpoint can only be invoked by user with Amplify authenticated/unauthenticated roles.  
+2. Logs can only be sent by mobile application users with Amplify authenticated/unauthenticated roles.
+3. The lambda invocation is only allowed from the provision AWS API Gateway resource.
+4. The S3 bucket containing the `remotelogginconstraints.json` can only be read by the Lambda execution role.
+
+### Replace the placeholder values with your own values:
+* `<CloudWatch-log-group-name>` is the log group that logs will be sent to.
```

Comment:
can we please link to the set up backend resources anchor link. And mention here that you would have set that up in that section.

Reply:
Added anchor link and additional clarification that the sample cdk is re-create the same resource that is already created.